mst/saml-okta-mst.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Setting up SAML authentication with Okta
+excerpt: Learn how to make corporate level configuration like the authentication setup on Managed Service for TimescaleDB 
+product: mst
+---
+
+# Set up authentication
+
+Security Assertion Markup Language (SAML) is a standard for exchanging authentication
+and authorization data between an identity provider and a service provider. You can operate
+it with MST so that you and your collaborators can use your company's favorite authentication service.
+
+## Before you begin
+
+1.  Create a new authentication method in MST.
+1.  Create the Okta application and assign users to the Okta application.
+
+<procedure>
+
+## Creating a new authentication method
+
+1.  Login to the [MST account] [mst-login].
+
+1.  Under `PROJECT` in the top left, click the drop down arrow and select `See all projects & accounts`.
+
+1.  Click the `Account` you want to edit or create a new one.
+
+1.  In the `Account page`, select the `Authentication` tab.
+
+1.  Click `Add Authentication Method` and set the `Method Name` as `<OKTA>` and
+    `Method Type` as `SAML`.
+
+1.  Choose the team to add invited people or leave it blank and click `Add`.
+
+1.  The two parameter `Metadata URL`, and `ACS URL` to setup the SAML
+    authentication in your Identity Provider appears.
+
+</procedure>
+
+## Creating the Okta application and assigning users
+
+This is a two step process. First create the SAML SP-Initiated
+authentication flow, then create a bookmark app that redirects to
+the Managed Service for TimescaleDB login page.
+
+<procedure>
+
+1.  Login to the `Admin` portal and navigate to the `Applications` tab.
+
+1.  Click `Add Application` and click `Create New App` You should see
+    the `Create a new Application Integration` form.
+
+1.  Select `SAML 2.0` for the `Sign on method`, then click `Create`.
+
+1.  In the `Create SAML Integration` set the `App name`, `App logo`, and `App visibility`,
+    then click `Next`.
+
+1.  Update the `Configure SAML` page with these values:
+    |Parameter|Example value|Description|
+   |-|-|-|
+   |Single sign on URL|`https://api.timescale.io/v1/sso/saml/account/{account_id}/method/{account_authentication_method_id/acs`|
+   The value in Timescale Console on the newly created `Authentication method` page.|
+   |Audience URI (SP Entity ID)|`https://api.timescale.io/v1/sso/saml/account/{account_id}/method/{account_authentication_method_id}/metadata`|
+   The value in Timescale Console on the newly created `Authentication method` page.|
+   |Default Relay State|https://portal.managed.timescale.com/| is the homepage
+   of the MST Console and is fundamental for IdP initiated sign-on to function
+   correctly.|
```

Comment:
Does the table display correctly? I've usually done this with a nested unordered list, like Step 6. here: https://docs.timescale.com/mst/latest/logging/#creating-a-loggly-service-integration